Source/WebCore/dom/Document.cpp

```diff
@@ -5419,7 +5419,9 @@ void Document::setWindowAttributeEventListener(const AtomString& eventType, cons
 
 void Document::dispatchWindowEvent(Event& event, EventTarget* target)
 {
+#if PLATFORM(IOS_FAMILY)
```

Comment:
This isn't right. We want to disable this assertion in macOS builds. Instead, we should modify the definition of isScriptAllowed to check isInWebProcess() as well. isScriptAllowed should return true when !isInWebProcess().